governing-documents/security.md

```diff
@@ -0,0 +1,266 @@
+agreed to follow.
+
+## Vulnerability Disclosure Process and Associated Rules
+
+All Hyperledger projects, including this project, follow the associated process
+and rules for vulnerability disclosures.  We note that this outline is derived
+from the OpenSSF,
+https://github.com/ossf/oss-vulnerability-guide/blob/main/maintainer-guide.md.
+
+Each project will have a security team (for more details, see the section on
+people infrastructure below).  The security team will be comprised of
+maintainers or contributors to the project who are knowledgeable about
+security and is responsible for responding to and helping to fix security
+vulnerabilities.
+
+The security team (see below for details) for this project will do the
+following for each reported vulnerability:
+1. Acknowledge receipt (see the section on “Report Intakes” for more) of the
+issue to the reporter within 2 business days.
+2. Assess the issue.  Engage with the reporter to ask any outstanding questions
+about the report and how to reproduce it.   If the report is not considered a
+vulnerability, then the reporter should be informed and this process can be
+halted. If the report is still a regular bug (just not a security
+vulnerability), the reporter should be informed (if necessary) of the regular
+process for reporting bugs, available <where your project has regular bug reports>.
+3. Some issues may require more time and resources to correct.  If this
+particular report is more complex, discuss an embargo period with the reporter.
+The embargo period should be negotiated with the reporter and must not be
+longer than 90 days.
+4. Create a patch for the issue (see the “Private Patch Deployment
+Infrastructure” section).
+5. Request a CVE for the issue (see the “CNA/CVE Reporting” section).
+6. Decide the date of public release.
+7. If applicable, Notify members of the embargo list of the upcoming patch
+and release, as described above.
+8. Cut a new (software) release in which the bug is fixed..
+9. Publicly disclose the issue within 48 hours after the release. It is
+recommended that this is done through Github security advisories (see the
+“(Github) Security Advisories” section for more details).
+
+## “People” Infrastructure
+
+This section details the required basic vulnerability disclosure infrastructure
+for the XYZ Project.  There are quite a few necessary pieces of infrastructure,
+so we go through them in detail here.
+
+**Security team:**  the XYZ Project must have a security response team of at
+least three maintainers.  This response team shall be set up BEFORE incidents
+happen so that people know who to contact and how to contact them when an
+emergency issue arises.  It can be difficult to track down someone with unique
+knowledge (e.g. in a particular area of cryptography) who is capable of
+fixing a problem in a short period of time.
+1. Each maintainer will be a member of the Hyperledger security email list
```

Comment:
I am assuming this is each maintainer of the security team and not each maintainer. If so, we should rephrase as:
```suggestion
1. Each security team member will be a member of the Hyperledger security email list
```